### PR TITLE
fix: always add COPY directives in DoCacheProbe

### PR DESCRIPTION
**Pull request by johnstcn**
Required in order to correctly handle RunCacheProbe. Without it, we will be unable to correctly detect the final layer of the built image.

### CODE REVIEW

File: options/options.go [hunk 1]
```diff
@@ -485,6 +497,9 @@ func (o *Options) Markdown() string {
 	_, _ = sb.WriteString("| - | - | - | - |\n")
 
 	for _, opt := range cliOptions {
+		if opt.Hidden {
```
Comment (johnstcn) [inline options/options.go:500]: review: drive-by; I don't think hidden options should be documented.

**Review by mafredri — APPROVED** [review]
